frappe/public/js/frappe/ui/filters/filter.js

```diff
@@ -52,6 +52,7 @@ frappe.ui.Filter = class {
 			"Markdown Editor": ["Between", "Timespan", ">", "<", ">=", "<=", "in", "not in"],
 			Password: ["Between", "Timespan", ">", "<", ">=", "<=", "in", "not in"],
 			Rating: ["like", "not like", "Between", "in", "not in", "Timespan"],
+			Float: ["like", "not like", "Between", "in", "not in", "Timespan"],
```

Comment:
between should work?

Reply:
I had the same thought, but it doesn't. I think it's currently specific to datetime fields.

Reply:
Ah db_query doesn't support. DB does.